### DISCUSSION TITLE
Tahoma2D Sample Data

### DISCUSSION THREAD

**Discussion by andeon** [tahoma2d/tahoma2d#1012] [Show and tell]
@manongjohn I'd like to discuss creating a sample for Tahoma2D. When I started porting my themes, I felt the lack of images to publicize the program. I really don't like using Dwanko to do this, so I started imagining a mascot for Tahoma.

The TD2 sample needs to cover all levels and with some types of animations, as I suggested in the image.

![TD2_sample_data](https://user-images.githubusercontent.com/11843239/176316928-6e097e3f-86e8-4e75-a0ad-a101f15e371f.png)

Ps. I believe people like more cute images; maybe I need to work on the young moose design, but generally it not functional without having a lot of shadows and lights. I have other simpler designs, but let's start here.

**Comment by manongjohn** [comment]
Yes, I did want to create sample scene for T2D similar to OT but was't sure what i wanted as a mascot.

As far as mascots go, l am leaning away from a "cartoony" moose.  I was thinking the character would be boy-ish with not-so-over-the-top moose-like features.  I personally like Japanese Anime, so would lean towards that style, perhaps adopting a chibi-like version of what I described.

It's something I wanted to initially create and then get feedback to see what works. Havent drawn in a long while so likely whatever i come up with will look terrible.🙁

**Reply by andeon** [reply]
Trying to design the character you idealized! I think we need to create shapes that will work properly in cut-out animation and also in plastic tool.

![TD2_sample_data_2](https://user-images.githubusercontent.com/11843239/176592780-891a265d-f643-4b10-9c16-c820b51e7dd5.png)

**Comment by andeon** [comment]
A creature like Netflix's Sweet Tooth? I don't remember if there is a name for this type of style in Japan, it's not Neko, because it's for anything like cat ears.

So something half human and half animal, but you'd like to put on a helmet or hat so you don't know if it's an outfit or a body part?

A chibi human version with small horns can also work very well.

About the sample, I think needs to have only the files that were used, a project for each level, and with folders and files well organized, the _Toonz Harlequin Traditional Workflow Tutorial_ and  the _Toonz Paperless Workflow Guide_ had a lot of stuff it doesn't help most people understand easily how the program works or use as a reference. _Opentoonz Sample_ has far fewer files, but specifically focus only in a studio workflow.

@manongjohn  Try to draw something, we have a lot of tools to improve it in vector or brushes. I had are many designs ready, but they were all "cartoony" moose 😁

**Comment by manongjohn** [comment]
Hmm...seems i might need to rethink this human-like character...looks a little too demonic 😄

**Reply by andeon** [reply]
Yeap, if we try to change the design to a pajama or hoodie, it will remain human and still contain the animal counterpart in the outfit.

**Reply by andeon** [reply]
Now trying new Chibi proportions (generally Chibis are 1.75~3 heads tall) and also mapping the cut out parts. Thinking about the animation structure, if we could put the cycle: Idle, Walk, Run and Jump or just the character acting something.

![TD2_sample_data_3](https://user-images.githubusercontent.com/11843239/177011826-0ea7970c-a060-4b73-b5a9-c483d6795569.png)

I was creating a basic "locked" animated puppet and using the **Stage Schematic** to connect the final parts of the character animation to each other, this can be reused indefinitely for any new character created. But it can make new users believe that they will always have to create a basic animation first. So I abandoned this idea, we need to keep focus to do it more clearly possible.

**Comment by artisteacher** [comment]
I think the antlers are too small in this last one. I like the young moose at the top of the discussion!

**Comment by artisteacher** [comment]
Also, maybe they would look less demon like if the antlers were more on the sides of the head/closer to the ears?

**Reply by andeon** [reply]
I need to finish a sketching costume/hoodie/pajama version to show Jhon. But now I'm thinking of glowing effects like energy for antlers, something like the character Code Sariel. 

I confess that my real interest in this is to have a good sample data to show what kind of workflow Tahoma2D offers. 

Still far away, because has a lot of things to do before starting the animation. I don't know if Jhon plans an open repository and also have no idea if the commits would work properly, but it would be very interesting after the release of the animation could improve it punctually, of course some limits would have to be stipulated.

**Comment by andeon** [comment]
Another sketch now drawing directly in Tahoma with the vector tool. Not sure if we need to stick to the same logo colors, about clothing, maybe a poncho? 
![TD2_sample_data_4](https://user-images.githubusercontent.com/11843239/207520801-95c991d7-fe01-43d1-9d0c-d6d955f44d74.png)


Adding the try with a poncho:

![TD2_sample_data_4_2](https://user-images.githubusercontent.com/11843239/207979932-80086e0b-ffb2-4833-b9c5-bb53d8a80645.png)

**Reply by artisteacher** [reply]
Cute! Love the poncho idea.

**Reply by manongjohn** [reply]
I kinda like it. It's in the direction I was thinking, and definitely not a demonic looking baby anymore. LOL.

Slight issue. It looks similar to Tama from the anime Natsume's Book of Friends  that i accidentally ran across today.

![image](https://user-images.githubusercontent.com/19245851/208334985-2de2f97a-1577-4656-a2aa-e4eb7b922c8d.png)

**Reply by andeon** [reply]
Wow, that Tama design could easily be a T2D mascot if this anime didn't exist. This height of it would be so cool, like an pixie or gnome in my mind I was just thinking about human height, precisely that of a boy and now that we know about it need avoiding the temptation to create something similar 😢

Was going to ask about quantity and type of antlers. Personality, colors you would like for it. Should we go more to tech, nature or mythical for the character's story?

**Comment by artisteacher** [comment]
I don't think they are similar enough to be a problem? Sure there are similarities, but the proportions, colors, and style are different enough to not be derivative in my opinion.

**Comment by manongjohn** [comment]
I think this character can still work. Tama is a dragon in the Anime where as Tahoma is a moose. To @artisteacher's point I think there may be enough differences that it can still work plus the horns are bigger and he has big ears

As for the character's story, I would lean towards either techy (it is animation software afterall) or mystical. Techno-mage, anyone? 🙂

Need to explore this more....and I need to learn how to draw it consistantly. Lol.

**Comment by andeon** [comment]
Thinking about Japanese animation and I remembered "Unico" by Osamu Tezuka, so I finally avoided the western moose cartoon and tried this one and liked the result. I was thinking about him being a shapeshifter like a familiar magical creature. The suit can have 5 or 7 folds, so when it transforms, some folds from the back come to the front covering the whole body.

![TD2_sample_data_5](https://user-images.githubusercontent.com/11843239/209723821-3547c23b-8c4e-40da-afb3-059ccb1d5db2.png)

I still haven't fixed the bad line strokes or painted correctly, but you get the idea

Maybe for animation: 

A digital pen falling in the middle of the white screen, the moose comes and catches it with his mouth. He start running drawing the background poorly, then staggers, rolls over and turns into a boy who fixes everything.

![Raf_anin](https://user-images.githubusercontent.com/11843239/209739000-72971b26-3604-498b-9773-527413f6bda3.png)

**Reply by artisteacher** [reply]
Cute idea!

**Reply by manongjohn** [reply]
Sorry for VERY late reply....either i never got the notification of this (seems to happen a lot with Discussion posts) or got so sidetracked that i forgot.  I had just released 1.4 earlier in the month. 

As much as I really like the human form design, I think we just created the chibi form of Sweet Tooth. Lol. 

I think i will abandon the human form idea, but your animal form design has some real potential as the actual mascot.  Would make a good plushie! 😄

May need a little tweeking so its not confused with a deer.

**Reply by andeon** [reply]
I'll redesign working on the antlers and snout. 

I also think it would be better if us create multiple samples instead of just one. That way, we cover more animation principles and Tahoma functions as well.

**Comment by andeon** [comment]
@manongjohn  I think I've finally got the design right, John. Let me know if you approve, so we can use it as a starting point.


![td2_sample_data_6](https://github.com/user-attachments/assets/75d89fa5-81ce-4bf4-a37b-20d08d26f973)

I'm not sure if these colors match what you have in mind. Let me know your thoughts!

![test_background_logo](https://github.com/user-attachments/assets/c0337fc5-d75f-47b0-bee2-ba12f2776b2d)

**Reply by artisteacher** [reply]
Cute!

**Reply by andeon** [reply]
Now, I’ve decided to modify the antlers to make them balanced and symmetrical, not just the antlers but the entire shape, ensuring the silhouette is visually pleasing, with the antlers enhancing the character’s design. This will help facilitate the animation by keeping it simple. I’ll upload a new version soon

I'm thinking about organizing folders and creating efficient project structures and files that could help understand the animation process and workflow, and make it easier for people to recreate or reuse assets when animating.

<div style="display: flex; justify-content: space-between;">

<div style="width: 45%;">

**Folder Structure 1:**

Tahoma2DProject/
├── +inputs/
├── +drawings/
├── +scenes/
├── +extras/
├── +outputs/
│   │   ├── assets/               
│   │   │   ├── modelSheets/      
│   │   │   ├── props/            
│   │   │   └── ...etc/ 
├── +palettes/
└── ProjectFile.tnz            
    

</div>

<div style="width: 45%;">

**Folder Structure 2:**


Tahoma2DProject/
├── animation/                     
│   ├── +inputs/                    
│   ├── +drawings/                  
│   ├── +scenes/                    
│   ├── +extras/                    
│   ├── +outputs/                   
│   ├── +palettes/  
├── assets/                       <-- All assets used for animation
│   ├── modelSheets/               <-- Model sheets
│   ├── props/                     <-- Props (objects, accessories)
│   ├── ...etc
└── ProjectFile.tnz  
</div>

</div>

@artisteacher  Do you have any suggestions for creating good structures? 

I'm using this as a reference https://tahoma2d.readthedocs.io/en/latest/managing_projects.html#adding-custom-default-folders

**Comment by andeon** [comment]
I'm still working on standardizing the appearance and poses. I'm also creating additional files, such as the model sheet, to complete the project. In this front pose, I tried covering it with a full dewlap, but I didn’t like the result. So, I’m going to keep the previous version short, like a collar. I think I’m going to move the antlers to the top of the head, either in front or parallel to the ears. I need work more in snout should be better defined with the help of shading, giving it a more appealing appearance.

![TD2_model_sheet_color](https://github.com/user-attachments/assets/f1f1885a-bde0-4764-9d55-8b7fa71f02a9)

**Comment by artisteacher** [comment]
I think people keep projects organized in a lot of different ways. I would try to keep it fairly simple.

**Comment by andeon** [comment]
My plan is to gradually add more files, so asset folders will be essential to store and support animations that will come later, such as Character Head Turnaround, walking, jumping, secondary actions, and so on. Therefore, I believe that Structure 2 is more suitable and easier to organize. By dividing the project this way, we can address the lack of references and materials for new users and, in the future, aim for a larger animation that could potentially serve as a demo reel.

ProjectT2D/
├── _assets/
├── animation/
│ │ ├── walking <-- Inside All default struct folders tahoma
│ │ │ ├── +inputs/ 
│ │ │ ├── +...
│ │ ├── jumping
│ │ │ ├── +inputs/
│ │ │ ├── +...

I created a paper layout that simulates  [AJAfr01](https://aja.gr.jp/jigyou/chousa/aja_layout), using some translated ref from [Sakuga](https://sakuga.fandom.com/wiki/Layout) and the measurements that I saw from Origins Animation Group, which I found to be quite effective for working with 1920x1080, where the canvas would be 2236x1582 without overloading weaker machines . I'm testing the violet color in Tahoma 2D for the borders and text of the template.


![Layout_paper](https://github.com/user-attachments/assets/fead4754-694a-462a-983b-686a1e1794d7)

**Comment by andeon** [comment]
Despite their differences, horses and moose share some similarities in movement. Both animals display similar walking and trotting patterns. Looking at the Second Book of the Tezuka School of Japanese Animation closely aligns with these motions. I'm now working to ensure that the design incorporates these movement characteristics to the fullest extent possible.

![horse_tezuka](https://github.com/user-attachments/assets/a1cb914b-8f7d-463d-933a-3ccff8ebd149)

But I'm still waiting for feedback from @manongjohn to make any changes he deems necessary.

**Comment by artisteacher** [comment]
That book looks helpful!
